deployment/terraform/dataspace/variables.tf

```diff
@@ -35,12 +35,17 @@ variable "application_sp_object_id" {
   description = "object id of application's service principal object"
 }
 
-variable "key_file" {
-  description = "name of a file containing the private key in PEM format"
+variable "key_file_registry" {
+  description = "name of a file containing the Registration Service private key in PEM format"
   default     = null
 }
 
-variable "public_key_jwk_file" {
-  description = "name of a file containing the public key in JWK format"
+variable "public_key_jwk_file_registry" {
```

Comment:
nit: minor but relates to the topic of `registry` vs `registration service`; ultimately this is up to you to decide (same as above)

Reply:
I renamed the `registry` to either `authority` for did document resources and 'registrationservice` for Registration Service resources

Reply:
I renamed the `registry` to either `authority` for did document resources and 'registrationservice` for Registration Service resources


> The DID and private key are related not just to "registration service", but really to "dataspace authority" which is more than that. Names should reflect this

indeed, I renamed the `registry` to either `authority` for did document resources and `registrationservice` for Registration Service resources

Reply:
I renamed the `registry` to either `authority` for did document resources and 'registrationservice` for Registration Service resources.